lib/plugin.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('mz/fs');
+const JSZip = require('jszip');
+const path = require('path');
+const semver = require('semver');
+const Changelog = require('./changelog');
+
+function readInfosFromComposerJson(composerJsonString) {
```

Comment:
_Information_ does not have a plural form:
```suggestion
function readInfoFromComposerJson(composerJsonString) {
```